src/core/tests/frontend/shared/test_builtin_extensions_1/test_extension.cpp

```diff
@@ -29,6 +30,11 @@ ov::OutputVector CustomTranslatorONNX(const ov::frontend::NodeContext& node) {
     return ov::OutputVector();
 }
 
+ov::OutputVector ReluToSwishTranslatorONNX(const ov::frontend::NodeContext& node) {
+    auto swish = std::make_shared<ov::opset8::Swish>(node.get_input(0));
```

Comment:
The segfault is thrown when `~Model` try to release memory allocated here (in a dll provided by a user)